### ISSUE TITLE
Bug: Problem Building Application under Windows 

### ISSUE THREAD

**Issue by Genei180** [mattermost/focalboard#1709]
## Steps to reproduce the behavior
I followed the Steps described under: https://www.focalboard.com/contribute/getting-started/dev-tips/

After Step 5. Hit F5 and select Go: Launch Server

The following Error Occurs for me on a Windows System:
`Unable to unlink socket.  "server/server.go:378"="The system cannot find the path specified."`

The file path that should be opened under windows after the Config File is: "/var/tmp/focalboard_local.socket"

## Expected behavior
To Build the Application for Debugging on 127.0.0.1:8000

## Edition and Platform
Forked from Github Commit: 690d424dfe28d3583634e8d7d7e59e365cac71d8

## Additional context
A Windows file Path gets Accepted if you Specify it in the Config, but Only if the File "focalboard_local.socket" has been created somewhere...

**Comment by Genei180** [comment]
I am already Working on a Fix, but it still needs to be Tested on Windows

**Comment by CodeCaterpillar666** [comment]
I have the same problem building application under MacOS.
`error [2021-11-17 10:36:58.845 -08:00] Unable to unlink socket.                 caller="server/server.go:378" error="no such file or directory"`

**Comment by chenilim** [comment]
Housekeeping old issues: The error message indicates a problem with the admin API, which is a file socket. The workaround is to set `enableLocalMode` in the `config.json` file and restart.

Closing this issue for now, but please reopen or file a new issue if anyone is still seeing this. cc @jespino, @ogi-m, @sbishel.
